### PR TITLE
Fluent adaptive

### PR DESCRIPTION
**Pull request by Juveniel**
part of: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/adaptive/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+                pointer-events: none;
+            }
+
+            .k-list-title,
+            .k-listgroup-title {
+                padding: 0;
+            }
+
+            .k-listgroup {
+                margin-inline: -#{map-get( $kendo-spacing, 2 )};
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/adaptive/_layout.scss:411]: take out the value in a variable and shouldn't this be calc( -1 * var)?

File: packages/fluent/scss/adaptive/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+            }
+        }
+
+        // Filter menu
+        .k-filter-menu {
+            padding: map-get( $kendo-spacing, 2 );
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/adaptive/_layout.scss:417]: take out the value in a variable

File: packages/fluent/scss/adaptive/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+            display: flex;
+            flex-flow: column nowrap;
+            gap: map-get( $kendo-spacing, 2 );
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/adaptive/_layout.scss:420]: take out the value in a variable

File: packages/fluent/scss/adaptive/_layout.scss [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+            }
+
+            .k-filter-tools {
+                display: flex;
+                flex-flow: row nowrap;
+                gap: inherit;
+            }
+
+            .k-listgroup {
+                margin-inline: -#{map-get( $kendo-spacing, 2 )};
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/adaptive/_layout.scss:440]: take out the value in a variable and calc( -1 * var)

File: packages/fluent/scss/adaptive/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+$kendo-adaptive-menu-bg: get-theme-color-var( primary-100 ) !default;
+$kendo-adaptive-menu-text: $white !default;
+
+$kendo-adaptive-menu-clear-text: get-theme-color-var( primary-100 ) !default;
+
+$kendo-adaptive-menu-item-border: var( --kendo-component-border, inherit ) !default;
+$kendo-adaptive-menu-title-text: var( --kendo-component-text, inherit ) !default;
+
+$kendo-adaptive-border-width: 1px !default;
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/adaptive/_variables.scss:20]: `kendo-spacing, hair`

**Review by PaskalevStoyan — CHANGES_REQUESTED** [review]

**Review by joneff — APPROVED** [review]
adaptive module will be reworked for 2022 R3 SP1

**Pull request merged**